From 12f8717110b15fc6b4b69935a9b41d3695c5d9c9 Mon Sep 17 00:00:00 2001
From: nagachika <nagachika@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
Date: Sun, 2 Mar 2014 15:25:22 +0000
Subject: [PATCH 1/5] merge revision(s) r45220:

	* README.EXT.ja: [DOC] Fix typo "macro macro" @utenmiki [Fixes GH-551]
	  https://github.com/ruby/ruby/pull/551


git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/branches/ruby_2_0_0@45248 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
---
 ChangeLog     | 5 +++++
 README.EXT.ja | 2 +-
 version.h     | 8 ++++----
 3 files changed, 10 insertions(+), 5 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index e61a82e..5789fa2 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+Mon Mar  3 00:24:38 2014  Zachary Scott  <e@zzak.io>
+
+	* README.EXT.ja: [DOC] Fix typo "macro macro" @utenmiki [Fixes GH-551]
+	  https://github.com/ruby/ruby/pull/551
+
 Sat Feb 22 22:11:58 2014  NAKAMURA Usaku  <usa@ruby-lang.org>
 
 	* ext/io/console/console.c (console_dev): need read access for conout$
diff --git a/README.EXT.ja b/README.EXT.ja
index 932a1c7..cc7eaf7 100644
--- a/README.EXT.ja
+++ b/README.EXT.ja
@@ -111,7 +111,7 @@ FIXNUMとNILに関してはより高速な判別マクロが用意されてい
 換結果がintのサイズに収まらない場合には例外を発生します．
 それから，FIXNUMに限らずRubyのデータを整数に変換する
 「NUM2INT()」および「NUM2LONG()」というマクロがあります．こ
-れらのマクロはマクロはデータタイプのチェック無しで使えます
+れらのマクロはデータタイプのチェック無しで使えます
 (整数に変換できない場合には例外が発生する)．同様にチェック無
 で使える変換マクロはdoubleを取り出す「NUM2DBL()」があります．
 
diff --git a/version.h b/version.h
index 79f93db..20f837a 100644
--- a/version.h
+++ b/version.h
@@ -1,10 +1,10 @@
 #define RUBY_VERSION "2.0.0"
-#define RUBY_RELEASE_DATE "2014-02-24"
-#define RUBY_PATCHLEVEL 451
+#define RUBY_RELEASE_DATE "2014-03-03"
+#define RUBY_PATCHLEVEL 452
 
 #define RUBY_RELEASE_YEAR 2014
-#define RUBY_RELEASE_MONTH 2
-#define RUBY_RELEASE_DAY 24
+#define RUBY_RELEASE_MONTH 3
+#define RUBY_RELEASE_DAY 3
 
 #include "ruby/version.h"
 
-- 
1.9.0

